specification/recoveryservicessiterecovery/resource-manager/Microsoft.RecoveryServices/stable/2021-02-10/service.json

```diff
@@ -13360,284 +14719,1310 @@
+          "format": "date-time",
+          "description": "The agent version expiry date.",
+          "type": "string",
+          "readOnly": true
+        },
+        "driverVersionExpiryDate": {
+          "format": "date-time",
+          "description": "The driver version expiry date.",
+          "type": "string",
+          "readOnly": true
+        },
+        "lastHeartbeatUtc": {
+          "format": "date-time",
+          "description": "The time of the last heartbeat recieved from the agent.",
+          "type": "string",
+          "readOnly": true
+        },
+        "reasonsBlockingUpgrade": {
+          "description": "The whether update is possible or not.",
+          "type": "array",
+          "items": {
+            "enum": [
+              "AlreadyOnLatestVersion",
+              "RebootRequired",
+              "AgentNoHeartbeat",
+              "RcmProxyNoHeartbeat",
+              "ProcessServerNoHeartbeat",
+              "IncompatibleApplianceVersion",
+              "NotProtected",
+              "UnsupportedProtectionScenario",
+              "DistroIsNotReported",
+              "DistroNotSupportedForUpgrade",
+              "MissingUpgradePath",
+              "InvalidAgentVersion",
+              "InvalidDriverVersion",
+              "Unknown"
+            ],
+            "type": "string",
+            "x-ms-enum": {
+              "name": "AgentUpgradeBlockedReason",
+              "modelAsString": true
+            }
+          },
+          "readOnly": true
+        },
+        "isUpgradeable": {
+          "description": "A value indicating whether agent is upgradeable or not.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "InMageRcmFailbackNicDetails": {
+      "description": "InMageRcmFailback NIC details.",
+      "type": "object",
+      "properties": {
+        "macAddress": {
+          "description": "The mac address.",
+          "type": "string",
+          "readOnly": true
+        },
+        "networkName": {
+          "description": "The network name.",
+          "type": "string",
+          "readOnly": true
+        },
+        "adapterType": {
+          "description": "The adapter type.",
+          "type": "string",
+          "readOnly": true
+        },
+        "sourceIpAddress": {
+          "description": "The IP address.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "InMageRcmFailbackPlannedFailoverProviderInput": {
+      "description": "Provider specific input for InMageRcmFailback failover.",
+      "required": [
+        "recoveryPointType"
+      ],
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/PlannedFailoverProviderSpecificFailoverInput"
+        }
+      ],
+      "properties": {
+        "recoveryPointType": {
+          "description": "The recovery point type. See SrsDataContract.RecoveryPoint enum for possible values.",
```

Comment:
Remove - _"See SrsDataContract.RecoveryPoint enum for possible values."_

Reply:
removed

---
In reply to: [588106542](https://github.com/AsrOneSdk/azure-rest-api-specs/pull/16#discussion_r588106542) [](ancestors = 588106542)